src/Maths/Silk.NET.Maths/Simd/Simd64.Constants.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#if INTRINSICS
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+
+namespace Silk.NET.Maths
+{
+    public static class Simd64<T> where T : unmanaged
+    {
```

Comment:
Shouldn't there be an IsHardwareAccelerated somewhere in this file?

Reply:
@HurricanKai that's what I wanted to ask about. I didn't get what this method is supposed to do

Reply:
Basically it's just a shorthand for "is one of the ISAs we use available" so usually `Sse(2).IsSupported || AdvSimd.IsSupported`

Reply:
Got it